ui/src/data/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -40,11 +40,13 @@ const DetailPage = ({
   const metadata = result.get('metadata');
   const title = metadata.getIn(['titles', 0]);
   const abstract = metadata.getIn(['abstracts', 0]);
+  const keywords = metadata.get('keywords', List());
```

Comment:
the component called is array safe, you can just do 
`  const keywords = metadata.get('keywords');`